### ISSUE TITLE
Add ability to translate language names in the language plugin

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#5411]
## Type of report

Feature request

## Provide description of the new feature

At the moment the default list of languages is hardcoded right inside the plugin → https://github.com/ckeditor/ckeditor4/blob/0150a16041546fdf71ca3313be87ad500f44c200/plugins/language/plugin.js#L26

However, the fact that all languages are defined by the ISO-639-1 codes means that it should be possible to provide translations for their names (Library of Congress provides [translations for English, French and German](https://www.loc.gov/standards/iso639-2/php/code_list.php)) in the plugin's lang files. It should be even possible to simplify the format of the [`config.language_list` configuration variable](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-language_list) from `<languageCode>:<languageLabel>[:<textDirection>]` to `<languageCode>[:<textDirection>]`.
